lib/test/karma.js

```diff
@@ -239,23 +239,26 @@ var yargsOptionDefiniton = {
     string:true,
   }
 };
-var checkKarmaReporter = yargs.createCheck()
-  .withGate(function (argv) {
-    return !argv.help;
-  })
-  .withTest({
-    karmareporter: function(value) {
-      if (typeof value !== 'undefined') {
-        try {
-          getKarmaReporterPluginPath(value);
-        }
-        catch (ex) {
-          return 'Illegal value for "reporter"\n' + ex;
-        }
+function checkKarmaReporter(argv) {
```

Comment:
since yargs is now dry, yargs options should now be decoupled from this implementation file (and promoted to the task file)